Sources/URBNAlert/AlertController.swift

```diff
@@ -14,7 +14,7 @@ public class AlertController: NSObject {
     private var alertIsVisible = false
     private var queue: [AlertViewController] = []
     private var alertWindow: UIWindow?
-    public var presentingWindow = UIApplication.shared.windows.first ?? UIWindow(frame: UIScreen.main.bounds)
+    public var presentingWindow = UIApplication.shared.keyWindow ?? UIWindow(frame: UIScreen.main.bounds)
```

Comment:
It looks like `keyWindow` is deprecated as of iOS12-13, are we sure this is the right path forward?
https://developer.apple.com/documentation/uikit/uiapplication/1622924-keywindow

Reply:
In an application with multiple, separate Windows (i.e. iPad OS) there might be some issues using `keyWindow`.

There'a few alternatives if we don't wan tot use the deprecated `var`.
```
UIApplication.shared.windows.first { $0.isKeyWindow }
```
source: https://stackoverflow.com/questions/57134259/how-to-resolve-keywindow-was-deprecated-in-ios-13-0

If we are concerned about finding the current top window for multiple connected scenes, we can do something like this.
```
extension UIApplication {
    var currentWindow: UIWindow? {
        connectedScenes
        .filter({$0.activationState == .foregroundActive})
        .map({$0 as? UIWindowScene})
        .compactMap({$0})
        .first?.windows
        .filter({$0.isKeyWindow}).first
    }
}
```
source: https://stackoverflow.com/questions/57009283/how-get-current-keywindow-equivalent-for-multi-window-scenedelegate-xcode-11

Reply:
Im cool with using this for the time being

Reply:
I’d prefer not to use a deprecated method. I’m fine with `currentWindow` extension, except I’d replace that last filter with a `.first(where:)`